documentation/build/maintainer.md

```diff
@@ -79,4 +86,27 @@ Edit the the `makefile` to make the following changes:
 3. Run `make docker_APSI_build`
 
 
+### Building only the ArangoML WebUI 
```

Comment:
Maybe match the TOC entry
`Building a frontend-only Docker image for the ArangoML WebUI`